src/main/java/beam/physsim/jdeqsim/cacc/roadcapacityadjustmentfunctions/RoadCapacityAdjustmentFunctionMock.java

```diff
@@ -0,0 +1,24 @@
+package beam.physsim.jdeqsim.cacc.roadcapacityadjustmentfunctions;
+
+import org.matsim.api.core.v01.network.Link;
+
+public class RoadCapacityAdjustmentFunctionMock implements RoadCapacityAdjustmentFunction {
```

Comment:
This should also be using an `Optional` over a mock

Reply:
Although this one has more functionality, so there is an argument for this, but I wouldn't call it a mock. Mocks are for testing, this is more of a Null Object. So if kept I would rename to `EmptyRoadCapacityAdjustmentFunction` or the like. And realistically it can be the default instead of having to be explicitly set.